hw/top_earlgrey/doc/design/README.md

```diff
@@ -540,6 +540,12 @@ The first EDN instance, `u_edn0` is intended to be configured to deliver entropy
 
 <!-- END CMDGEN -->
 
+#### Open drain / virtual open drain
+
+Pads of type *BidirOd* have true hardware support for open drain termination: when *open drain* is enabled in pinmux's attributes for such a pad, the PMOS transistor that would otherwise pull the output high for a logic 1 is disabled and there is no parallel connection to the supply rail; thus for a logic 1 the pad then has an impedance equal to the output impedance of the CMOS transistor pair (while for a logic 0 the pad is still pulled low by the NMOS transistor).
+Pads of type *BidirStd* instead support a virtual open drain termination: when *virtual open drain* is enabled in pinmux's attributes for such a pad, the PMOS transistor is disabled but a parallel internal resistor still connects the pad output to the supply rail; thus for a logic 1 the pad has an impedance equal to that of the internal pull-up resistor.
```

Comment:
Could you please clarify how this pull-up resistor relate to PULL_EN and PULL_SELECT of the inmux PAD_ATTR registers?
The proposed description makes it sound as if you get a pull-up regardless of PULL_EN.